tests/scaffoldings/dotd/docker-compose.yaml

```diff
@@ -10,9 +10,8 @@ services:
     environment:
       PGUSER: another_odoo
       PGPASSWORD: anotherodoopassword
-      PGHOST: postgresql
```

Comment:
I think some entrypoint script will fail if you set an invalid `$PGHOST` anyway.

I think I will explain better the test I expected:

1. Add to `environment`: 
   ```yaml
   ODOO_LIMIT_REQUEST: 8000
   ODOO_LIST_DB: 0
   ```
1. Add a `dotd/custom/conf.d/request_limit.conf` with:
   ```
   [options]
   limit_request = 9000
   ```
1. Add `dotd/custom/entrypoint.d/600-goodconf.py`:

   ```python
   #!/usr/local/bin/python-odoo-shell
   from odoo.tools import config
   assert config.get("limit_request") == 9000
   assert not config.get("list_db")
   ```

This way we make sure the env variables get into config, and that manual configs override them.